package.json

```diff
@@ -75,6 +75,7 @@
         "@langchain/community": "^0.0.54",
         "@langchain/openai": "^0.0.28",
         "@mui/icons-material": "^5.10.9",
+        "@mui/joy": "^5.0.0-beta.48",
```

Comment:
question: why are we using Joy MUI? That will introduce a third library. We use the default Material UI component library https://mui.com/material-ui/getting-started/

Reply:
I used Joy MUI because it was suggested for the activity, and I found it appropriate to implement it in the project. This library, being a more recent version within the MUI ecosystem, incorporates more modern and updated concepts. Additionally, Joy MUI allows us to handle requirements that could be more complex to implement with traditional Material UI, helping to minimize rework in future changes and making code maintenance easier

Reply:
Let's record in a Architectural Decision Record and discuss why should we keep both. It seem that there's [official documentation](https://mui.com/joy-ui/integrations/material-ui/) about pros and cons from MUI.